staging/prometheus-operator/templates/mesosphere-hooks/prometheus-get-cluster-id.yaml

```diff
@@ -0,0 +1,26 @@
+{{- if .Values.grafana.enabled }}
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ .Values.mesosphereResources.hooks.prometheus.jobName }}
+  namespace: {{ .Release.Namespace }}
+  annotations:
+    "helm.sh/hook": pre-install,pre-upgrade
+    "helm.sh/hook-weight": "5"
+    "helm.sh/hook-delete-policy": hook-succeeded,before-hook-creation
+spec:
+  template:
+    metadata:
+      name: {{ .Values.mesosphereResources.hooks.prometheus.jobName }}
+    spec:
+      containers:
+        - name: kubectl
+          image: bitnami/kubectl:1.16.2
```

Comment:
do we want to templatize image? if we are using `kubectl` at different places  within this chart then templating would help in case we have to update the image in future

Reply:
I think I'm going to hold off on this because it would require modifying the helpers file which is an upstream file (trying to avoid adding more changes to the fork). We can templatize once we separate the upstream chart out.

Reply:
I thought everything under `mesosphere-hooks` is in our control, no?